lib/mix/pages/introduction-to-mix.md

```diff
@@ -0,0 +1,320 @@
+# Introduction to Mix
+
+In this guide, we will learn how to build a complete Elixir application, with its own supervision tree, configuration, tests, and more.
+
+The requirements for this guide are (see `elixir -v`):
+
+  * Elixir 1.11.0 onwards
+  * Erlang/OTP 22 onwards
+
+The application works as a distributed key-value store. We are going to organize key-value pairs into buckets and distribute those buckets across multiple nodes. We will also build a simple client that allows us to connect to any of those nodes and send requests such as:
+
+```text
+CREATE shopping
+OK
+
+PUT shopping milk 1
+OK
+
+PUT shopping eggs 3
+OK
+
+GET shopping milk
+1
+OK
+
+DELETE shopping eggs
+OK
+```
+
+In order to build our key-value application, we are going to use three main tools:
+
+* ***OTP*** _(Open Telecom Platform)_ is a set of libraries that ships with Erlang. Erlang developers use OTP to build robust, fault-tolerant applications. In this chapter we will explore how many aspects from OTP integrate with Elixir, including supervision trees, event managers and more;
+
+* ***[Mix](`Mix`)*** is a build tool that ships with Elixir that provides tasks for creating, compiling, testing your application, managing its dependencies and much more;
+
+* ***[ExUnit](`ExUnit`)*** is a test-unit based framework that ships with Elixir.
+
+In this chapter, we will create our first project using Mix and explore different features in <abbr title="Open Telecom Platform">OTP</abbr>, Mix, and ExUnit as we go.
+
+<b>Fuck</b>
```

Comment:
![image](https://github.com/elixir-lang/elixir/assets/9582/eee2c238-2edc-4423-8aea-09b9a133b3c9)

Reply:
Ahahaha holy crap, I kept this in here to check inline HTML 😄